embedded-google-pubsub/src/main/java/com/playtika/test/pubsub/PubSubResourcesGenerator.java

```diff
@@ -54,18 +63,28 @@ private void createTopicAndSubscription(TopicAndSubscription ts) {
         createTopic(ts.getTopic());
 
         if (ts.getSubscription() != null) {
-            createSubscription(ts.getTopic(), ts.getSubscription());
+            createSubscription(ts.getTopic(), ts.getSubscription(), ts.getDeadLetter());
         }
     }
 
-    public Subscription createSubscription(String topicName, String subscriptionName) {
+    public Subscription createSubscription(String topicName, String subscriptionName, DeadLetter deadLetter) {
         ProjectTopicName topic = ProjectTopicName.of(projectId, topicName);
         ProjectSubscriptionName subscription = ProjectSubscriptionName.of(projectId, subscriptionName);
 
         try {
             log.info("Creating subscription: {}", subscription);
-            return subscriptionAdminClient
-                    .createSubscription(subscription, topic, PushConfig.getDefaultInstance(), 100);
+
+            Builder builder = Subscription.newBuilder()
```

Comment:
Previously subscription was created with additional params `PushConfig.getDefaultInstance(), 100`. Can you please check what do they stand for and whether they are needed now?

Reply:
It seems that those additional parameters were provided, because there is no other overloaded method that would allow you skip those (except the one I used). To stay compatible I restored the ack deadline setting (set to 100), but it seems no sense to provide `PushConfig.getDefaultInstance()` as it does configure nothing. In the end it resolves to:
```
  private PushConfig() {
    pushEndpoint_ = "";
  }
  ```